.github/workflows/test.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         swiftver:
           - swift:5.5
           - swift:5.6
+          - swift:5.7
           - swiftlang/swift:nightly-main
         swiftos:
           - focal
```

Comment:
If you're gonna bump this, might as well grab the full update I did on my branch locally:
```diff
diff --git a/.github/workflows/test.yml b/.github/workflows/test.yml
index c688128..1c60cb0 100644
--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -1,44 +1,42 @@
 name: test
-on:
+on:
   - pull_request
-defaults:
-  run:
-    shell: bash
+env:
+  LOG_LEVEL: debug
+  SWIFT_DETERMINISTIC_HASHING: 1
+  MYSQL_DATABASE: 'test_database'
+  MYSQL_DATABASE_A: 'test_database'
+  MYSQL_DATABASE_B: 'test_database'
+  MYSQL_USERNAME: 'test_username'
+  MYSQL_USERNAME_A: 'test_username'
+  MYSQL_USERNAME_B: 'test_username'
+  MYSQL_PASSWORD: 'test_password'
+  MYSQL_PASSWORD_A: 'test_password'
+  MYSQL_PASSWORD_B: 'test_password'
+  MYSQL_HOSTNAME: 'mysql-a'
+  MYSQL_HOSTNAME_A: 'mysql-a'
+  MYSQL_HOSTNAME_B: 'mysql-b'
+
 jobs:
   linux-all:
+    if: github.event.pull_request.draft == false
     strategy:
       fail-fast: false
       matrix:
         dbimage:
           - mysql:5.7
           - mysql:8.0
-          - mariadb:10.3
-          - mariadb:10.7
+          - mariadb:10.4
+          - mariadb:10.11
           - percona:8.0
         swiftver:
-          - swift:5.5
-          - swift:5.6
-          - swiftlang/swift:nightly-main
-        swiftos:
-          - focal
-    container: ${{ format('{0}-{1}', matrix.swiftver, matrix.swiftos) }}
+          - swift:5.5-bionic
+          - swift:5.6-focal
+          - swift:5.7-jammy
+          - swiftlang/swift:nightly-5.8-jammy
+          - swiftlang/swift:nightly-main-jammy
+    container: ${{ matrix.swiftver }}
     runs-on: ubuntu-latest
-    env:
-      LOG_LEVEL: debug
-      # Unfortunately, fluent-mysql-driver details leak through here
-      MYSQL_DATABASE: 'test_database'
-      MYSQL_DATABASE_A: 'test_database'
-      MYSQL_DATABASE_B: 'test_database'
-      MYSQL_USER: 'test_username'
-      MYSQL_USERNAME: 'test_username'
-      MYSQL_USERNAME_A: 'test_username'
-      MYSQL_USERNAME_B: 'test_username'
-      MYSQL_PASSWORD: 'test_password'
-      MYSQL_PASSWORD_A: 'test_password'
-      MYSQL_PASSWORD_B: 'test_password'
-      MYSQL_HOSTNAME: 'mysql-a'
-      MYSQL_HOSTNAME_A: 'mysql-a'
-      MYSQL_HOSTNAME_B: 'mysql-b'
     services:
       mysql-a:
         image: ${{ matrix.dbimage }}
@@ -76,6 +74,7 @@ jobs:
         run: swift test --package-path fluent-mysql-driver

   macos-all:
+    if: github.event.pull_request.draft == false
     strategy:
       fail-fast: false
       matrix:
@@ -96,9 +95,6 @@ jobs:
     runs-on: ${{ matrix.macos }}
     env:
       LOG_LEVEL: debug
-      MYSQL_DATABASE: 'test_database'
-      MYSQL_USERNAME: 'test_username'
-      MYSQL_PASSWORD: 'test_password'
       MYSQL_HOSTNAME: '127.0.0.1'
     steps:
       - name: Select latest available Xcode
@@ -106,6 +102,7 @@ jobs:
         with:
           xcode-version: ${{ matrix.xcode }}
       - name: Install MySQL, setup DB and auth, and wait for server start
+        shell: bash
         run: |
           brew install ${{ matrix.dbimage }} && brew link --force ${{ matrix.dbimage }}
           brew services start ${{ matrix.dbimage }}
```